core/include/drivers/serial.h

```diff
@@ -40,4 +44,25 @@ struct serial_ops {
 	int (*getchar)(struct serial_chip *chip);
 };
 
+struct io_pava {
+	paddr_t pa;
+	vaddr_t va;
+};
```

Comment:
True, but I did not think it was worth creating a new file just for this. Where would you put this code? It's only used in serial drivers currently, I thought we'll move it if/when we want to use it elsewhere.

OK for `s/io_pava/io_pa_va`.